source/features/new-repo-disable-projects-and-wikis.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import domLoaded from 'dom-loaded';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import {getRepoURL} from '../github-helpers';
+
+async function disableWikiAndProjects(): Promise<void> {
+	delete sessionStorage.rghNewRepo;
+
+	void api.v3(`repos/${getRepoURL()}`, {
+		method: 'PATCH',
+		body: {
+			has_projects: false,
+			has_wiki: false
+		}
+	});
+	await domLoaded;
+	const wiki = document.evaluate('//*[@data-content=\'Wiki\']/ancestor-or-self::*[@class=\'d-flex\']', document, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue;
+	wiki?.remove();
+	const projects = document.evaluate('//*[@data-content=\'Projects\']/ancestor-or-self::*[@class=\'d-flex\']', document, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue;
+	projects?.remove();
```

Comment:
Maybe it's not worth it even with a helper function… the equivalent is not that long in the first place:
```suggestion
	select('[data-content="Wiki"]')?.closest('.d-flex')!.remove();
	select('[data-content="Projects"]')?.closest('.d-flex')!.remove();
```

Perhaps XPath comes handy when we must select elements and text node by content, which requires multiple lines of code with selectors _or_ we just use `.firstChild` and hope for the best (which isn't safe)